man/gssproxy.conf.5.xml

```diff
@@ -216,7 +216,7 @@
                             At level 3 and above, KRB5_TRACE output is logged.
                             If KRB5_TRACE was already set in the execution
                             environment, trace output is sent to its value
-                            instead.
+				instead. Refer to docs/KRB5_TRACE.md for more information.
```

Comment:
Looks like indentation issue here.

Reply:
When I had added a line it did the weird indent and I did not go back and fix. Corrected now, but it seems the 'i' is a space too far to the left in `$ git show`.

I was not too sure the best way to preview the XML file outside of a text editor to make sure that it was actually structured appropriately.

Reply:
not a big deal